packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1555,6 +1555,8 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
 /// pass a [GlobalKey] to the constructor and use [GlobalKey.currentState] to
 /// save or reset the form field.
 ///
+/// The `value` parameter maps to [FormField.initialValue].
```

Comment:
You maybe can do `[value]`, but defer to y'all (I don't update user-facing docs often and don't know the considerations).

Reply:
I see both [] and `` are used for constructor parameters. Let me double check!

Reply:
Should be backticks for this case as it is not a field in `DropdownButtonFormField`:) https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements